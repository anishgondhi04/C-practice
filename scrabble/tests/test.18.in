10
4
2 3 H hello
2 2 H hello
2 4 H hello
7 2 V hello
0

