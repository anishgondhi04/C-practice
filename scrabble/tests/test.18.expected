Invalid word placement: (2,2) H, hello
Invalid word placement: (2,4) H, hello
Invalid word placement: (7,2) V, hello
..........
..........
..........
..hello...
..........
..........
..........
..........
..........
..........
